client/src/views/ChallengeView.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+    <div id="header-container" class="text-white">
+        <h1 class="text-md-h1 mt-10 mb-12 font-weight-medium" id="title">PICK A CHALLENGE</h1>
+
+        <div class="ma-10">
+            <h2 style="margin-bottom: 1vh;" class="text-h6 font-weight-bold">Choose your challenge to complete over the 12 days of the
```

Comment:
We'll change the description properly later but for now please change 12 days to 14 days and recalculate the km per day for each challenge accordingly

Reply:
Km per day get funky, we'll come back this